extensions/shared/blocks/cover/index.js

```diff
@@ -13,29 +13,13 @@ import { addFilter } from '@wordpress/hooks';
 import coverEditMediaPlaceholder from './cover-media-placeholder';
 import coverMediaReplaceFlow from './cover-replace-control-button';
 import jetpackCoverBlockEdit from './edit';
-import { isUpgradable } from './utils';
 import './editor.scss';
 
-const addVideoUploadPlanCheck = ( settings, name ) => {
-	if ( ! settings.isDeprecation && isUpgradable( name ) ) {
```

Comment:
Since `isUpgradable` is removed now, how is this functionality being handled? Or is it not handled for now and that's a follow-up PR?

Reply:
it's handled inside the hooked function. All functions check the block name at the top of them. Something similar to the following:

```es6
const { name } = useBlockEditContext();
if ( 'core/cover' !== name ) {
	return <OriginalComponent { ...props } />;
}
```

Reply:
Actually I forgot to push the commit, Jerry. Addressed ea2ed5d7963c064fa87bd75a042750ea5bbc72b6